src/include/catalog/catalog_entry/catalog_entry.h

```diff
@@ -66,7 +66,10 @@ class KUZU_API CatalogEntry {
     virtual void serialize(common::Serializer& serializer) const;
     static std::unique_ptr<CatalogEntry> deserialize(common::Deserializer& deserializer);
 
-    virtual std::string toCypher(main::ClientContext* /*clientContext*/) const { KU_UNREACHABLE; }
+    virtual std::string toCypher(main::ClientContext* /*clientContext*/,
+        std::string /*exportPath*/) const {
```

Comment:
I'm slightly concerned about this API change because in the future if some other catalog entry requires a new information we will have to change the base interface and touches many files.

I would try to follow the design of `TableBindData` or `xxxAuxiliaryxxx` to have an empty base class here and let which other CatalogEntry implement its own information.